packages/search/tests/dummy/app/styles/app.less

```diff
@@ -0,0 +1,2 @@
+@import 'navi-core.less';
```

Comment:
did i miss the core pr? did you move styling over there?

Reply:
I ended up removing styling and using styling from core, like the `@denali-color-brand-600`

Reply:
👍